### ISSUE TITLE
训练时进度条不动

### ISSUE THREAD

**Issue by shelly-xue** [MVIG-SJTU/AlphaPose#959]
您好，我用的linux16.04，训练卡在epoch 0，进度条一直是0，请问大家知道是什么原因吗？非常感谢

**Comment by Fang-Haoshu** [comment]
ctrl c看看有没有上面报错

**Comment by shelly-xue** [comment]
@Fang-Haoshu 感谢您的回复。这个是Ctrl c后的报错内容。您知道原因是啥吗？我的环境是pytorch=1.7 python=3.6
File "scripts/train.py", line 58, in train
    for i, (inps, labels, label_masks, _, bboxes) in enumerate(train_loader):
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/site-packages/tqdm-4.62.3-py3.6.egg/tqdm/std.py", line 1180, in __iter__
    for obj in iterable:
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 435, in __next__
    data = self._next_data()
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 1068, in _next_data
    idx, data = self._get_data()
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 1034, in _get_data
    success, data = self._try_get_data()
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/site-packages/torch/utils/data/dataloader.py", line 872, in _try_get_data
    data = self._data_queue.get(timeout=timeout)
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/multiprocessing/queues.py", line 104, in get
    if timeout < 0 or not self._poll(timeout):
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/multiprocessing/connection.py", line 257, in poll
    return self._poll(timeout)
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/multiprocessing/connection.py", line 414, in _poll
    r = wait([self], timeout)
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/multiprocessing/connection.py", line 911, in wait
    ready = selector.select(timeout)
  File "/home/jhdu/anaconda3/envs/alphapose_cam/lib/python3.6/selectors.py", line 376, in select
    fd_event_list = self._poll.poll(timeout)
KeyboardInterrupt

**Comment by Fang-Haoshu** [comment]
这个好像看不太出来，试试dataloader的worker数量加大？比如gpu数量的4倍
